NEWS.md

```diff
@@ -1,5 +1,6 @@
 ## rtables 0.6.2.9001
  * Cleaned up spelling in documentation ([#685](https://github.com/insightsengineering/rtables/issues/685))
+ * Added decimal alignment test. Custom appearance vignette and alignment checks have been updated with modifications coming from `formatters`.
```

Comment:
I don't think we should mention added tests in the NEWS file, please edit this so it just mentions the custom appearance vignette and alignment checks (either as one entry or two) and be a little more detailed about those changes